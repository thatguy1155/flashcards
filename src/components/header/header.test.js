import { render } from '@testing-library/react';
import TestRenderer from 'react-test-renderer';
import Header from './header';

describe('Header', () => {
  test('non-shallow render', () => {
    // eslint-disable-next-line new-cap
    const element = new TestRenderer.create(
      <Header />,
    );
    expect(element.toJSON()).toMatchSnapshot();
  });

  it('says Flash Cards', () => {
    const header = render(<Header />);
    const msg = header.getByText('Flash Cards');
    expect(msg).toBeInTheDocument(1);
  });
});
