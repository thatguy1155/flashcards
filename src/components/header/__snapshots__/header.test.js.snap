// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header non-shallow render 1`] = `
<div
  className="header"
>
  <h1>
    Flash Cards
  </h1>
  <div
    className="buttons"
  >
    <button>
      review
    </button>
    <button>
      add
    </button>
  </div>
</div>
`;
